crates/mpc-protocols/src/mpc_service.rs

```diff
@@ -37,13 +39,15 @@ enum MPCStatus {
 pub struct MPCService {
     mpc_instances: Mutex<HashMap<ObjectID, MPCInstance>>,
     pending: Mutex<VecDeque<ObjectID>>,
+    active_instances_counter: Mutex<usize>,
```

Comment:
I needed it to be Mutex because the MPCService is a field within the AuthorityState that is being used across different threads. Anyway, I removed all those mutexes as I changed the entire MPCService field in the authoritystate to be a mutex.